control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller_test.go

```diff
@@ -2096,6 +2118,10 @@ func (*fakeReleaseProvider) ComponentVersions() (map[string]string, error) {
 	return map[string]string{}, nil
 }
 
+func (*fakeReleaseProvider) ComponentImages() map[string]string {
```

Comment:
Do we still need this?

Reply:
Yes because now it's part of the interface